features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/async/RecordedDataQueueHandler.kt

```diff
@@ -107,9 +114,10 @@ internal class RecordedDataQueueHandler {
         val rumContextData = rumContextDataHandler.createRumContextData()
             ?: return null
 
-        // if the view changed then clear the drawable cache
+        // if the view changed then clear the drawable cache and bitmap pool
         if (rumContextData.prevRumContext != rumContextData.newRumContext) {
             cache.clear()
+            bitmapPool.clear()
```

Comment:
do we need to clear it here ? What if those bitmaps can be reused for the next View ?